osmosis-1/osmosis.zone_assets.json

```diff
@@ -4850,6 +4850,16 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/uwu.sol.pica.png"
         }
       }
+    },
+    {
+      "chain_name": "osmosis",
+      "base_denom": "factory/osmo104jtrwcljnxfljhml8mxrw7qetcsdmqvy3sprw/ucosmousd",
+      "categories": [
+        "rwa"
```

Comment:
this doesn't seem to be rwa, but maybe stablecoin (would also need the 'peg_mechanism' property here and the 'traces' at the chain registry pointing back to forex::USD)

Reply:
CosmoUSD has nothing to do with currencies, it's RWA because it's the points in the Cosmopolitan Ultimate Sports Club.